packages/loader/container-loader/src/container.ts

```diff
@@ -752,7 +752,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         // runtime matches pending ops to successful ones by clientId and client seq num, so we need to close the
         // container at the same time we get pending state, otherwise this container could reconnect and resubmit with
         // a new clientId and a future container using stale pending state without the new clientId would resubmit them
-        this._deltaManager.close();
+        this.forceReadonly(true);
```

Comment:
I'm not sure that's correct. @wes-carlson - FYI

Reply:
Please see my comment. @wes-carlson is in the loop.

Reply:
Andrei is right. Previously we were getting the pending states after runtime was disposed. That isn't possible if the pending states are cleared on disposal.